tekton/resources/nightly-tests/pipeline-deploy-test-s390x-template.yaml

```diff
@@ -160,7 +166,7 @@ spec:
           - name: package
             value: github.com/tektoncd/pipeline
           - name: container-registry
-            value: $(params.container-registry)
+            value: "192.168.0.13:6001"
```

Comment:
Why is this value hardcoded here now? Is the container registry not specified in the pipeline run?

Reply:
The error code suggests that an HTTP request was sent over an HTTPS port (443):

```
        error: no objects passed to apply
        Error: error processing import paths in "./config/controller.yaml": error resolving image references: writing sbom: GET http://s390x-cluster.bastion-z.svc.cluster.local:443/v2/: unexpected status code 400 Bad Request: <html>
        <head><title>400 The plain HTTP request was sent to HTTPS port</title></head>
        <body>
```

It does look like an issue on client side rather than on the registry.

Reply:
yes right, client is sending image with http request i.e `http://s390x-cluster.bastion-z.svc.cluster.local:443/v2/` we have registry configured with `https`, let me check pipeline run, to pass https registry.